valhalla/tyr/serializers.h

```diff
@@ -16,6 +16,7 @@
 #include <valhalla/meili/match_result.h>
 #include <valhalla/midgard/gridded_data.h>
 #include <valhalla/proto/api.pb.h>
+#include <valhalla/proto/info.pb.h>
```

Comment:
again superfluous

Reply:
This has been fixed in the latest commit.